lib/mbta_v3_api/schedule.ex

```diff
@@ -17,7 +17,8 @@ defmodule MBTAV3API.Schedule do
 
   Util.declare_enum(
     :stop_edge_type,
-    Util.enum_values(:index, [:regular, :unavailable, :call_agency, :coordinate_with_driver])
+    Util.enum_values(:index, [:regular, :unavailable, :call_agency, :coordinate_with_driver]),
+    :call_agency
```

Comment:
question: This is described in the swagger docs as "Must phone agency to arrange pickup" which doesn't seem like a great default to me (though I don't think we use this anywhere client-side right now). Since this field isn't nullable and has well defined types, why not fall back to `CrashOnUnknownEvenThoughThisIsABadIdea` for any required / well defined GTFS field like is done already for `Route`? I could also see that being applied to `Stop.location_type` and `Vehicle.current_status`